app/services/foods_service.rb

```diff
@@ -0,0 +1,7 @@
+class FoodsService
+
+  def self.foods_search(ingredient)
+    response = Faraday.get("https://api.nal.usda.gov/fdc/v1/foods/search?api_key=#{ENV['API_KEY']}&query=#{ingredient}")
```

Comment:
I would rather see you set your query params in a block. This improves readability.